locales/en.json

```diff
@@ -1,10 +1,10 @@
 {
     "Title": "Contoso Air",
   "Home": {
-    "HeroMessage": "Fly to the sky."
+    "HeroMessage": "Crawl, walk run."
```

Comment:
This is a super funny April Fools day joke!

Reply:
Sure working on that.